src/packages/cli/src/__tests__/doctor.test.ts

```diff
@@ -24,7 +24,7 @@ it('should fail when db is missing', async () => {
 it('should fail when prisma schema is missing', async () => {
   const result = Doctor.new().parse([])
   await expect(result).rejects.toThrowErrorMatchingInlineSnapshot(
-    `Either provide --schema or make sure that you are in a folder with a schema.prisma file.`,
+    `Either provide --schema or configure a path in your package.json in a \`prisma.schema\` field or make sure that you are in a folder with a schema.prisma file.`,
```

Comment:
If you have a better wording here let me know

Reply:
Is this the full error message, or is there more around this sentence?